src/Core/Content/Media/Thumbnail/ThumbnailService.php

```diff
@@ -462,6 +462,15 @@ private function writeThumbnail($thumbnail, MediaEntity $media, string $url, int
             case 'image/jpeg':
                 imagejpeg($thumbnail, null, $quality);
 
+                break;
+            case 'image/webp':
+                if (function_exists('imagewebp')) {
+                    imagewebp($thumbnail, null, $quality);
+                }
+                else {
+                    throw new ThumbnailCouldNotBeSavedException($url);
+                }
```

Comment:
If you don't mind, can I make a suggestion here to remove the unnecessary `else` statement?
```suggestion
                if (!function_exists('imagewebp')) {
                    throw new ThumbnailCouldNotBeSavedException($url);
                }

               imagewebp($thumbnail, null, $quality);
```

Reply:
Sure